src/riscv/lib/src/pvm/tezos.rs

```diff
@@ -10,6 +10,9 @@ use ed25519_dalek::Signature;
 use ed25519_dalek::Signer;
 use ed25519_dalek::SigningKey;
 use ed25519_dalek::VerifyingKey;
+
+use libsecp256k1::{Message, PublicKey, Signature as SecpSig};
+use sha3::{Digest, Keccak256};
```

Comment:
Looks like [you need to run the formatter](https://github.com/tezos/riscv-pvm/actions/runs/15705636743/job/44252813366?pr=93#step:10:17). Try `scripts/format.sh --all`.